### ISSUE TITLE
Error in "Check for New Comments"

### ISSUE THREAD

**Issue by rdlauer** [orbit-love/community-ruby-linkedin-orbit#10]
First off - thanks for creating this integration, it'll be super helpful (when I can get it running!).

I'm about 99% sure I have everything set up correctly (have the two secrets created, the LinkedIn developer app, and the Heroku instance all set up). However, in the "Check for New Comments" workflow, I hit the following error. If anyone has any ideas I would appreciate the help!

```
Run linkedin_orbit --check-comments
  linkedin_orbit --check-comments
  shell: /usr/bin/bash -e {0}
  env:
    LINKEDIN_ORGANIZATION: ***
    LINKEDIN_TOKEN: ***
    ORBIT_API_KEY: ***
    ORBIT_WORKSPACE_ID: ***
Checking for new LinkedIn comments and posting them to your Orbit workspace...
/opt/hostedtoolcache/Ruby/2.7.2/x64/lib/ruby/gems/2.7.0/gems/linkedin_orbit-0.1.3/lib/linkedin_orbit/linkedin.rb:51:in `get_posts': undefined method `each' for nil:NilClass (NoMethodError)
	from /opt/hostedtoolcache/Ruby/2.7.2/x64/lib/ruby/gems/2.7.0/gems/linkedin_orbit-0.1.3/lib/linkedin_orbit/linkedin.rb:13:in `process_comments'
	from /opt/hostedtoolcache/Ruby/2.7.2/x64/lib/ruby/gems/2.7.0/gems/linkedin_orbit-0.1.3/lib/linkedin_orbit/client.rb:58:in `comments'
	from /opt/hostedtoolcache/Ruby/2.7.2/x64/lib/ruby/gems/2.7.0/gems/linkedin_orbit-0.1.3/scripts/check_comments.rb:13:in `render'
	from /opt/hostedtoolcache/Ruby/2.7.2/x64/lib/ruby/gems/2.7.0/gems/thor-1.1.0/lib/thor/command.rb:27:in `run'
	from /opt/hostedtoolcache/Ruby/2.7.2/x64/lib/ruby/gems/2.7.0/gems/thor-1.1.0/lib/thor/invocation.rb:127:in `invoke_command'
	from /opt/hostedtoolcache/Ruby/2.7.2/x64/lib/ruby/gems/2.7.0/gems/thor-1.1.0/lib/thor.rb:392:in `dispatch'
	from /opt/hostedtoolcache/Ruby/2.7.2/x64/lib/ruby/gems/2.7.0/gems/thor-1.1.0/lib/thor/base.rb:485:in `start'
	from /opt/hostedtoolcache/Ruby/2.7.2/x64/lib/ruby/gems/2.7.0/gems/linkedin_orbit-0.1.3/bin/linkedin_orbit:26:in `<top (required)>'
	from /opt/hostedtoolcache/Ruby/2.7.2/x64/bin/linkedin_orbit:23:in `load'
	from /opt/hostedtoolcache/Ruby/2.7.2/x64/bin/linkedin_orbit:23:in `<main>'
Error: Process completed with exit code 1.
```

**Comment by hummusonrails** [comment]
Hey there @rdlauer!

I'll take a look at this soon and hopefully push out a new release to fix it soon.

Thanks for raising it!

Ben

**Comment by hummusonrails** [comment]
I believe this has been fixed in #11.

If there is an error message returned from LinkedIn during the request for an organization's posts that will be returned instead of trying to iterate on a non-iterable object. Hopefully, that error message can help diagnose if it is an issue with the LinkedIn credentials or whatnot. If there are simply no posts identified, the application will return that instead as well.
